HACKING.md

```diff
@@ -75,30 +75,15 @@ SENTRY_DSN=<DSN_FROM_ABOVE>
 
 The application will be reporting errors to your `sentry.io` project from now on.
 
-
-## Building CSS
```

Comment:
Why have you removed this section? It's important to know how to watch when "Hacking"

Reply:
Because when you run `dotrun` watch is done automatically. Do think it should be explicitly specified? If so should we also add how to build JS aswell?

Reply:
In that case its fine :+1: